src/main/resources/assets/groovyscript/lang/en_us.lang

```diff
@@ -1189,6 +1189,116 @@ groovyscript.wiki.naturesaura.spawning.entity.value=Sets the entity spawned
 groovyscript.wiki.naturesaura.spawning.removeByEntity=Removes all Altar of Birthing recipes that summon the given entity
 groovyscript.wiki.naturesaura.spawning.removeByName=Removes the Altar of Birthing recipe with the given name
 
+# ProdigyTech
+groovyscript.wiki.prodigytech.atomic_reshaper.title=Atomic Reshaper
+groovyscript.wiki.prodigytech.atomic_reshaper.description=Uses hot air and Primordium to convert items, can have a weighted random based output
+groovyscript.wiki.prodigytech.atomic_reshaper.time.value=The time needed to perform the recipe, in ticks. By default, takes the value from the ProdigyTech's config
```

Comment:
this lang key will generate the following line on the wiki:
```
- `int`. The time needed to perform the recipe, in ticks. By default, takes the value from the ProdigyTech's config. Requires greater than or equal to 1. (Default `-1`).
```
the `defaultValue` prop in the `@Property` annotation is supposed to take the role of providing an identical way to identify what the default value is, and so it essentially duplicates this information, which is confusing.
i would suggest implementing the fields and lang keys similar to how it is done for Extended Crafting:

https://github.com/CleanroomMC/GroovyScript/blob/master/src/main/java/com/cleanroommc/groovyscript/compat/mods/extendedcrafting/CombinationCrafting.java#L119
https://cleanroommc.com/groovy-script/mods/extendedcrafting/combination_crafting#recipe-builder

as this generates a line that looks like this:
```
- `int`. Sets the maximum amount of RF consumed per tick until the entire cost is paid. Requires greater than or equal to 0. (Default `ModConfig.confCraftingCoreRFRate`).
```

Reply:
ty for the tip, will do